phpstan.neon

```diff
@@ -15,3 +15,5 @@ parameters:
       param_type: 59
       property_type: 0 # We can't use property types until we support PHP 7.4.
       print_suggestions: false
+  typeAliases:
+    Asset_Info: 'array{ dependencies: array<string>, version: string }'
```

Comment:
In `PHPStan` we can either define the type on classes or globally on config files. Since it is a util fuction which is not wrapped by class so therefore we define the type globally.